integrations/Klaviyo/browser.js

```diff
@@ -57,20 +57,26 @@ class Klaviyo {
       return;
     }
     let payload = {
-      $id: message.userId || message.anonymousId,
+      $id: message.userId || message.context.userId || message.anonymousId,
       $email: message.context.traits.email,
       $phone_number: message.context.traits.phone,
-      $first_name: message.context.traits.firstName,
-      $last_name: message.context.traits.lastName,
+      $first_name:
+        message.context.traits.firstName ||
+        message.context.traits.firstname ||
+        message.context.traits.first_name,
+      $last_name:
```

Comment:
same as above

Reply:
also let's move these logic to utils for now?
something like getDefinedTraits(message) ==> return {email, phone, lastName, firstName, ..etc}